modules/doc/content/newsletter/2025/2025_01.md

```diff
@@ -14,16 +9,93 @@ convergence and divergence criteria. The parsed expression may include other
+
+- a logarithmic grid in enthalpy can now be used for properties computed from specific volume and specific enthalpy
+- a specific volume, specific energy tabulation can be read, but still rely on specific volume, specific enthalpy conversions to a pressure and temperature variable sets
+  in order to avoid an iterative solve
+- the number of out-of-bounds evaluations can now be tallied and reported in the console log
+
+### Navier Stokes
+
+The [VolumetricFlowRate.md] object was extended to support the linear finite volume discretization.
+
+An option to compute enthalpy directly rather than its approximation by the product of specific heat by temperature
+has been added in the [INSFVEnthalpyFunctorMaterial.md]. This facilitates the problem setup when the approximation is not
+valid, such as for compressible gases.
+
+The PIMPLE (PISO+SIMPLE) algorithm has been implemented. With this, the linear finite volume capabilities support
+transient simulations as well. For more information, see [PIMPLE.md].
+
+### Reactor module
+
+The [ControlDrumMeshGenerator.md] is now integrated in the Reactor Mesh Generator Building (RGMB) workflow. This abbreviated
+syntax facilitates the use of the mesh generators in the Reactor module to build Cartesian and hexagonal lattice-based reactor cores.
+It sets all the defaults and additional metadata that are expected by the downstream reactor physics applications (such as Griffin).
+
+### Solid Mechanics
+
+The [QuasiStaticSolidMechanicsPhysics.md] is no longer creating displacement variables without the block restriction of the `Physics`.
+This is intended for being able to restrict the solid mechanics equations to only part of the subdomain. Simulations using multiple solid
+mechanics `Physics` will automatically create the displacement variables using the union of all the block restrictions, if `add_variables` is
+set in the shared `[Physics/SolidMechanics/QuasiStatic]` block. Simulations with other `Physics` deriving or creating the `QuasiStaticSolidMechanicsPhysics`
+programmatically (such as in BISON) should likely create the displacement variables in a `Variables` block.
+
+The method used by the four-node shell elements ([ShellElements.md]) to define local coordinate systems has been revised. In the revised version, the user provides a reference direction for the first local coordinate, and that is then projected onto the plane of the shells and used as the first local coordinate direction. An AuxKernel has been added to visualize the local coordinate directions. These changes, together with the correction of a longstanding bug with the calculation of the normal direction, enable the use of these elements for accurate computation of the response of curved shells, as confirmed by multiple regression tests that agree well with published analytical solutions.
+
+The [PressureAction.md] is now controllable. It can be activated and de-activated using the [Controls system](syntax/Control/index.md).
```

Comment:
```suggestion
The [PressureAction.md] is now controllable. It can be activated and de-activated using the [Controls system](syntax/Controls/index.md).
```